sphinxlint.py

```diff
@@ -438,25 +521,31 @@ def parse_args(argv=None):
     if argv is None:
         argv = sys.argv
     parser = argparse.ArgumentParser(description=__doc__)
+
+    enabled_checkers_names = {
+        checker.name for checker in checkers.values() if checker.enabled
+    }
+
+    class EnableAction(argparse.Action):
+        def __call__(self, parser, namespace, values, option_string=None):
+            if values == "all":
+                enabled_checkers_names.update(set(checkers.keys()))
+            else:
+                enabled_checkers_names.update(values.split(","))
+
+    class DisableAction(argparse.Action):
+        def __call__(self, parser, namespace, values, option_string=None):
+            if values == "all":
+                enabled_checkers_names.clear()
+            else:
+                enabled_checkers_names.difference_update(values.split(","))
```

Comment:
Can't this be done more easily after the call to `parse_args`, instead of defining two classes?

Reply:
I don't think so. but if you can keep the current semantic with simplier code, I'll gladly accept it.

I think there's no way in argparse to keep the order, so those two would look the same:

- `--disable all --enable foo`
- `--enable foo --disable all`

in the current implementation, the first one means "just foo" and the second one means "nothing", without knowing the order of arguments it won't be doable (I bet).

The current implementation allows for things like:

    --disable all --enable foo,bar --disable bar

To enable just `foo`, it could be usefull in some contexts when some defaults arguments are governed by a thing and there's an ability to add more to tweak a bit ore.